cfme/common/vm.py

```diff
@@ -390,11 +391,12 @@ def load_details(self, refresh=False):
             sel.click(self.find_quadicon())
         else:
             if refresh:
-                # bz1392139 for 5.7, should be fixed in 5.7.1 - dajo
-                if version.current_version() < "5.7":
-                    toolbar.refresh()
-                else:
+                # bz1389299 for 5.7, should be fixed in 5.7.1 - dajo
+                reload_bug = BZ(1329299)
+                if reload_bug.bugzilla.get_bug(1329299).is_opened:
```

Comment:
Just a quick question, does a ```sel.refresh()``` work here? We could use that in both cases and avoid the BZ completely, just a thought, pretty quick to test/change